### ISSUE TITLE
cx_Freeze: Python error in main script

### ISSUE THREAD

**Issue by Ristelle** [Kramoule/happypanda#1]
Hey there! Thanks for taking this on, much appreciated!

When I try to launch Happypanda.exe I get:

---------------------------
cx_Freeze: Python error in main script
---------------------------
Traceback (most recent call last):
  File "C:\Users\Kramoule\AppData\Local\Programs\Python\Python35\lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 14, in run
  File "C:\Users\Kramoule\AppData\Local\Programs\Python\Python35\lib\site-packages\cx_Freeze\initscripts\Console.py", line 26, in run
  File "version\main.py", line 22, in <module>
  File "E:\Repo\happypanda\version\database\db.py", line 15, in <module>
  File "C:\Users\Kramoule\AppData\Local\Programs\Python\Python35\lib\sqlite3\__init__.py", line 23, in <module>
  File "C:\Users\Kramoule\AppData\Local\Programs\Python\Python35\lib\sqlite3\dbapi2.py", line 27, in <module>
ImportError: DLL load failed: The specified module could not be found.

---------------------------
OK   
---------------------------

Seems to be trying to call files specifically from your machine, I believe.
I don't see any logs generated, but let me know if you need other info.

Thanks!

**Comment by Kramoule** [comment]
Thanks for the feedback. I knew I should have test it in a new environment.
For now I'll remove the exe from the release until it's fixed.

**Comment by Kramoule** [comment]
@Ristelle I can't seem to reproduce this exact bug.
But it seems that it can't find **sqlite3.dll**. Can you try to move the **sqlite3.dll** from the main folder to the **lib** folder and tell me if it works ? (I personally had to move the **VCRUNTIME140.dll** from the lib folder to the main folder). 
Or download this zip: ~~[Happypanda-1.1.1.zip](https://github.com/Kramoule/happypanda/files/3005481/Happypanda-1.1.1.zip)~~ (try first the zip file in the next message)
What Windows OS are you using ?

**Comment by Kramoule** [comment]
I think I found the problem. It's related to my packaging software and the bug was fixed in a beta release anthony-tuininga/cx_Freeze#296
I updated my packager software. Try this one: [Happypanda-1.1.1_cx6.0b.zip](https://github.com/Kramoule/happypanda/files/3005618/Happypanda-1.1.1_cx6.0b.zip)

**Comment by Ristelle** [comment]
I'm running Windows 10 on my desktop. Looks like I'm getting the same kind of error with the new version. I also tried moving the dlls as you suggested with the same result.
However, I tried on my laptop which is still running Windows 8.1 and it worked right away. It even pulled the EH metadata correctly when I tried that! (Good job!)

Error message from Win10:
---------------------------
cx_Freeze: Python error in main script
---------------------------
Traceback (most recent call last):
  File "C:\Users\Kramoule\AppData\Local\Programs\Python\Python35\lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 14, in run
  File "C:\Users\Kramoule\AppData\Local\Programs\Python\Python35\lib\site-packages\cx_Freeze\initscripts\Console.py", line 26, in run
  File "version\main.py", line 18, in <module>
ImportError: DLL load failed: The specified module could not be found.

---------------------------
OK   
---------------------------

**Comment by Kramoule** [comment]
Strange because I have Win 10 on my laptop where I tested it and can run it without problem. I can have the exact same error as you only if I remove a specific dll (see below).

Glad that it at least works on one other computer than mine :)

Let's do some basic checks just to be sure:
- Did you downloaded the last zip I sent (**Happypanda-1.1.1_cx6.0b.zip**) ?
- Did you extract it in a new location ?
- Did you have these files/folder when extracted ?
![happypanda_tree](https://user-images.githubusercontent.com/22647002/54960720-23f09400-4f5e-11e9-96ac-1a3934f49d81.png)

I can only reproduce your error if I delete or move **Qt5Widgets.dll**
![happypanda_error1](https://user-images.githubusercontent.com/22647002/54960792-6ca84d00-4f5e-11e9-9be1-cf4a7a5d97b3.png)

**Comment by Ristelle** [comment]
> Did you downloaded the last zip I sent (Happypanda-1.1.1_cx6.0b.zip) ?

Yup.

> Did you extract it in a new location ?

Yeah, I tried several locations - across both of my drives just to be sure.

> Did you have these files/folder when extracted ?

Looks the same to me
![Capture](https://user-images.githubusercontent.com/48893249/54961196-b75da100-4f2d-11e9-8763-b257f25ce1ee.PNG)

--

I feel really dumb for not trying this sooner, but I just tried running as administrator and it started right up. I always forget that Win10 likes to ignore that I am the only admin on the system...  

Everything looks to be working properly now though!
